### PR TITLE
Release 1.10.1, release-next to main

### PR DESCRIPTION
**Pull request by mary-dcouto**

### CODE REVIEW

File: DesktopEdge/Models/ZitiIdentity.cs [hunk 1]
```diff
@@ -54,16 +65,25 @@ public ZitiIdentity()
...
 
+			if (!zid.IsMFAEnabled) zid.IsTimingOut = false;
+
 			if (id.Services != null) {
 				foreach (var svc in id.Services) {
 					if (svc != null) {
 						var zsvc = new ZitiService(svc);
 						zid.Services.Add(zsvc);
+						if (zid.IsMFAEnabled) {
+							if (zsvc.TimeoutRemaining>-1 && zsvc.TimeoutRemaining<1260) zid.IsTimingOut = true;
```
Comment (dovholuknf) [inline DesktopEdge/Models/ZitiIdentity.cs:85]: 1260??? 21 minutes? this is strange

Comment (dovholuknf) [reply DesktopEdge/Models/ZitiIdentity.cs:85]: same comment wrt all the other "1260" values

File: DesktopEdge/packages.config [hunk 2]
```diff
@@ -1,6 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<packages>
-  <package id="Newtonsoft.Json" version="12.0.3" targetFramework="net48" />
```
Comment (dovholuknf) [inline DesktopEdge/packages.config:3]: were these removed on purpose? i would expect the DesktopEdge to need these

File: ZitiDesktopEdge.Client/ServiceClient/AbstractClient.cs [hunk 3]
```diff
@@ -194,7 +194,7 @@ public abstract class AbstractClient {
 #if DEBUG
             Logger.Debug(msg);
 #else
-            Logger.Trace(msg);
+            Logger.Info(msg);
```
Comment (dovholuknf) [inline ZitiDesktopEdge.Client/ServiceClient/AbstractClient.cs:197]: please fix this - it should not be at Info for sure

File: ZitiUpdateService/checkers/GithubCheck.cs [hunk 4]
```diff
@@ -38,7 +45,7 @@ internal class GithubCheck : UpdateCheck {
 			return downloadFileName;
 		}
 
-		override public int IsUpdateAvailable(Version currentVersion) {
+		override public void IsUpdateAvailable(Version currentVersion, out int avail, out string publishedDate) {
```
Comment (dovholuknf) [inline ZitiUpdateService/checkers/GithubCheck.cs:48]: small nit: Maybe change the function name to reflect more accurately the purpose... just "CheckUpdate" or something?

File: DesktopEdge/Models/ZitiIdentity.cs (hunk 1 above)
Comment (mary-dcouto) [reply DesktopEdge/Models/ZitiIdentity.cs:85]: Jeremy, Is this zsvc.TimeoutRemaining field updated in the UI? If the user set the frequency to 20 minutes, the first notification may come between 40-20 minutes, second notification may come between 20-0 minutes.

File: DesktopEdge/packages.config (hunk 2 above)
Comment (mary-dcouto) [reply DesktopEdge/packages.config:3]: Jeremy upgraded the package config. @JeremyTellier  Can you answer this question?

File: DesktopEdge/Models/ZitiIdentity.cs (hunk 1 above)
Comment (JeremyTellier) [reply DesktopEdge/Models/ZitiIdentity.cs:85]: It should be 1200 for 20 minute to show the little icon next to the Service count. 

The service notification is going away right? If we want the UI to handle all the timeout logic for this those notifications will just mess up the sate of things so they are commented out in the next commit

File: DesktopEdge/packages.config (hunk 2 above)
Comment (JeremyTellier) [reply DesktopEdge/packages.config:3]: Package are not handled that way anymore that is legacy

**Review by JeremyTellier — APPROVED** [review]

**Pull request merged**